editoast/src/modelsv2/projects.rs

```diff
@@ -0,0 +1,241 @@
+use crate::error::Result;
+use crate::modelsv2::Save;
+use crate::views::pagination::{Paginate, PaginatedResponse};
+use crate::DbPool;
+use actix_web::web::Data;
+use async_trait::async_trait;
+use chrono::{NaiveDateTime, Utc};
+use derivative::Derivative;
+use diesel::sql_types::BigInt;
+use diesel::{sql_query, ExpressionMethods, QueryDsl};
+use diesel_async::{AsyncPgConnection as PgConnection, RunQueryDsl};
+use editoast_derive::ModelV2;
+use serde::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
+use crate::models::List;
+
+crate::schemas! {
+    Ordering,
+    ProjectWithStudies,
+    Project,
+    ProjectRow,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize, Derivative, ModelV2, ToSchema)]
+#[derivative(Default)]
+#[model(table = crate::tables::project)]
+#[model(row(derive(Debug, Clone, Serialize, ToSchema)))]
```

Comment:
```suggestion
```

You probably don't need any of those since `ProjectRow` is not meant to be used directly.